release_docs/RELEASE.txt

```diff
@@ -139,6 +139,13 @@ New Features
 
       (ADB - 2022/06/22)
 
+    - The perf tool has been removed
```

Comment:
This should goto the top of the section.

Reply:
Fixed